### DISCUSSION TITLE
1.5 控制

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#105] [Announcements]
# 1.5 控制

SICP Python 中文翻译

https://composingprograms.netlify.app/1/5

**Comment by lakerswillwin** [comment]
very goooood

**Comment by wangding** [comment]
命令行运行 doctest 的方式：
- 加上 `-v` 参数更妙：`python3 -m doctest -v <python_source_file>`
- 不加 `-v` 参数，单元测试通过后，没有任何提示信息。只有测试不通过时，才有提示信息。
